.pre-commit-config.yaml

```diff
@@ -9,11 +9,18 @@ repos:
     hooks:
       - id: isort
         name: isort (python)
+  - repo: https://github.com/pycqa/flake8
+    rev: 4.0.1
+    hooks:
+      - id: flake8
+        additional_dependencies:
+          - 'flake8-bugbear==21.11.29'
+          - 'flake8-pyi==22.5.1'
```

Comment:
Could you add a comment here and in the requirements file that they should be kept in sync?

Reply:
I added these in the same style as the existing comments in `requirements-tests.txt`. We can tweak the text if it's too pithy.